draft-ietf-taps-interface.md

```diff
@@ -1925,6 +1916,12 @@ Group. As noted in {{groups}}, this property is not entangled when Connections
 are cloned.
 
 
+### Niceness (Message)
+
+Integer Message Property - see {{msg-niceness}}.
+This property overrides the connection niceness - see {{conn-niceness}} - on a per-message basis.
```

Comment:
Mhh, I don't think this is true.
I think per-Connection Niceness and per-Message Niceness are different Properties and refer to different things (the priority of Connections within a Connection group, and the priority of Messages within a Connection), so they cannot override each other.
I'd be in favor of removing this sentence.

Reply:
Sorry, forgot about that - hope the new sentence captures the discussion.
Just removing it does will bring us to new discussions about this sooner or later.

Reply:
Good point. I think like this it's fine.